dead-simple-k8s/hostIP.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+sleep 10
+curl -s "${K8S_API:-localhost:8001}/api/v1/namespaces/$NS/pods/$POD_NAME" | jq '.status.hostIP' | sed 's/"//g'
```

Comment:
FWIW, I think this could be even more useful as an init/wrapper script:

i.e. `proxied_run`
```sh
#!/bin/sh

set -e

if [ $# -lt 1 ]; then
  echo "usage: proxied_run cmd args..." >&2
  exit 1
fi

k8s_api="${K8S_API:-localhost:8001}/api/v1/namespaces/$NS/pods/$POD_NAME"

host_ip=$(curl -s  | jq '.status.hostIP' | sed 's/"//g')
export http_proxy=$host_ip:4140 HTTP_PROXY=$host_ip:4140

exec "$@"
```